src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -365,12 +365,22 @@ private function loadProxyClient(ContainerBuilder $container, XmlFileLoader $loa
      */
     private function createHttpDispatcherDefinition(ContainerBuilder $container, array $config, $serviceName)
     {
-        foreach ($config['servers'] as $url) {
+        if (is_string($config['servers'])) {
             $usedEnvs = [];
-            $container->resolveEnvPlaceholders($url, null, $usedEnvs);
+            $container->resolveEnvPlaceholders($config['servers'], null, $usedEnvs);
```

Comment:
does this allow to specify a list of urls in a single env variable? or should we split on `,`? (though `,` is technically a legal part of a url, in practice this is a list of ips or host names, so it would be strange to have paths with `,` in it)